### ISSUE TITLE
comments in showdef?

### ISSUE THREAD

**Issue by antranigv** [norayr/voc#29]
showdef is an amazing tool, specially for newbies, it helps to understand the content of a module without reading the source code.
however, it will be more useful, it the .sym files where also commented about every TYPE, CONST, PROCEDURE, etc.
This also means that we will need to re-implement showdef, AFAIK it does not support comments.
thanks in advance.

**Comment by dcwbrown** [comment]
The Oakwood guidelines propose that showdef retain comments that begin `(**`. i.e. two asterisks instead of one.

**Comment by antranigv** [comment]
True, and, AFAIK, there is a tool implemented for that, called Watson. we should try either porting watson, and/or add an option for voc that creates .sym files with exported comments.
